packages/sdk/README.md

```diff
@@ -0,0 +1,10 @@
+# Zowe Native Protocol SDK
+
+The Zowe Native Protocol SDK is a TypeScript library to interact with the Zowe Native Protocol, which provides a native integration with z/OS systems over SSH, allowing developers to interact with mainframe resources directly from their application.
+
+## Building from source
+
+1. From the root of this repository, run `npm install` to install all the dependencies
+2. Run `npm run build` to build the SDK
+
+The SDK is compiled and saved in the `dist` folder.
```

Comment:
The SDK output would be in the `packages/sdk/lib` folder

Reply:
Whoops, forgot to update this text 😅